### DISCUSSION TITLE
Project for the Franciscan Office

### DISCUSSION THREAD

**Discussion by hcblassingame** [DivinumOfficium/divinum-officium#3261] [Ideas]
First of all, I'd like to express my appreciation and admiration for the work that has already been put into this.

Secondly, has anyone thought of undertaking a project to implement the Franciscan breviary in the project? In the main, it's the same as the Roman breviary, but there are different Propers, along with a different calendar.

I would be happy to work on this to contribute to the project, along with any API work that might be needed, but would appreciate any pointers for getting started that might be offered.

/henry

**Comment by mbab** [comment]
Good start point is to try read the code, it can be hard task at it is not well structured.
Make fork of repo, create branch:
- do addition in `web/www/horas/horas.dialog`
- in `web/www/horas/Latin/Tabulae` subdir `Kalendaria` create calendar file and make addition to `data.txt`
- add propers files with suffix `f` to `Sancti`, `Tempora`, `Commune` directories (eg. 10-04f.txt) 
- ask questions

Sorry for late answer I lived in belief that new Discussions post should be visible in Notifications.

**Reply by hcblassingame** [reply]
Not to be outdone in tardiness, thank you for your response! :)

I have created my own fork of the repo and am beginning to pull together what I will need to get started.

/henry

**Reply by FAJ-Munich** [reply]
As you might be able to see from the progress made on the Cistercian office, the idea of putting everything in the same folder as the Benedictine (I.e. SantiM etc.) and using conditionals proved to cause more issues than it helped. 

Therefore, it is advisable to create separate folders and use full file inclusions where appropriate. 

The biggest task is to get the rubrics on occurrence and concurrence right which might need some coding to be done. So an early assessment is necessary which Roman or Monastic rubrical structure (Trid, Divino, 196x) is the closest to build upon. 

And as @mbab has pointed out: ask questions; even the "silly" ones
